.rubocop_todo.yml

```diff
@@ -0,0 +1,841 @@
+# SupportedStyles: always, conditionals
+Style/AndOr:
+  Exclude:
+    - 'lib/hanami/configuration.rb'
+    - 'lib/hanami/generators/database_config.rb'
+    - 'lib/hanami/rendering_policy.rb'
+
+# Offense count: 6
+Style/AsciiComments:
+  Exclude:
+    - 'lib/hanami/application.rb'
+    - 'lib/hanami/environment.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+Style/BlockComments:
+  Exclude:
+    - 'test/fixtures/commands/application/new_app/spec/spec_helper.rspec.rb'
+    - 'test/fixtures/commands/application/new_container/spec/spec_helper.rspec.rb'
+
+# Offense count: 28
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, ProceduralMethods, FunctionalMethods, IgnoredMethods.
+# SupportedStyles: line_count_based, semantic, braces_for_chaining
+# ProceduralMethods: benchmark, bm, bmbm, create, each_with_object, measure, new, realtime, tap, with_object
+# FunctionalMethods: let, let!, subject, watch
+# IgnoredMethods: lambda, proc, it
+Style/BlockDelimiters:
+  Exclude:
+    - 'lib/hanami/commands/console.rb'
+    - 'lib/hanami/commands/server.rb'
+    - 'lib/hanami/environment.rb'
+    - 'test/cli_test.rb'
+    - 'test/commands/console_test.rb'
+    - 'test/commands/generate/action_test.rb'
+    - 'test/commands/generate/app_test.rb'
+    - 'test/commands/generate/mailer_test.rb'
+    - 'test/commands/generate/migration_test.rb'
+    - 'test/commands/generate/model_test.rb'
+    - 'test/configuration_test.rb'
+    - 'test/fixtures/config.ru'
+    - 'test/integration/head_request_test.rb'
+    - 'test/test_helper.rb'
+
+# Offense count: 28
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: braces, no_braces, context_dependent
+Style/BracesAroundHashParameters:
+  Exclude:
+    - 'lib/hanami/hanamirc.rb'
+    - 'lib/hanami/loader.rb'
+    - 'test/config/cookies_test.rb'
+    - 'test/configuration_test.rb'
+    - 'test/fixtures.rb'
+    - 'test/integration/body_parsers_test.rb'
+    - 'test/integration/container_middleware_test.rb'
+    - 'test/integration/cookies_test.rb'
+    - 'test/integration/sessions_test.rb'
+    - 'test/integration/streaming_test.rb'
+    - 'test/loader_test.rb'
+
+# Offense count: 81
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: nested, compact
+Style/ClassAndModuleChildren:
+  Enabled: false
+
+# Offense count: 4
+Style/ClassVars:
+  Exclude:
+    - 'lib/hanami/application.rb'
+    - 'lib/hanami/container.rb'
+    - 'test/application_test.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: Keywords.
+# Keywords: TODO, FIXME, OPTIMIZE, HACK, REVIEW
+Style/CommentAnnotation:
+  Exclude:
+    - 'lib/hanami/commands/generate/action.rb'
+
+# Offense count: 2
+Style/ConstantName:
+  Exclude:
+    - 'test/commands/console_test.rb'
+    - 'test/hanami_setup_test.rb'
+
+# Offense count: 36
+Style/Documentation:
+  Enabled: false
```

Comment:
Why disable it when most of the code is correctly documented? Having methods and classes documented is something the hanami team seem to care a lot about.